python/sdssdb/sqlalchemy/archive/sas.py

```diff
@@ -0,0 +1,142 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under a 3-clause BSD license.
+#
+# @Author: Joel Brownstein
+# @Date:   2019-08-01 06:54:15
+# @Last modified by: N Benjamin Murphy (n.benjamin.murphy@astro.utah.edu)
+# @Date:   2019-09-04 16:31:00
+
+from __future__ import absolute_import, division, print_function
+
+from sdssdb.sqlalchemy.archive import database, ArchiveBase, sas
```

Comment:
Many imports here are marked as not used by the linter. Please remove them.

Reply:
fixed.